app/src/main/java/org/fossasia/openevent/app/module/event/list/EventsPresenter.java

```diff
@@ -41,6 +43,16 @@ public void start() {
         return events;
     }
 
+    @Override
+    public void sortByName(boolean sortBy) {
```

Comment:
Here, the criteria should be passed instead of boolean and we are done

Reply:
@iamareebjamal but in the above comment you are asking to pass boolean only

Reply:
That is in adapter